inference-engine/src/transformations/include/transformations/op_conversions/convert_gather_0d.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API ConvertGather0D;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/*
+ * Description:
```

Comment:
Please use doxygen comments format
```
/**
 * @ingroup ie_transformation_common_api
 * @brief This transformation converts opset1::Gather to legacy GatherIE
 *        GatherIE takes axes as value and if indices input has empty shape (scalar)
 *        we unsqueeze indices input and squeeze GatherIE output.
 */
```

Reply:
And actually this transformation does nothing with GatherIE so please change description.

Reply:
Updated